src/main/groovy/com/sequenceiq/cloud/azure/client/AzureClient.groovy

```diff
@@ -371,7 +371,7 @@ class AzureClient extends RESTClient {
         } catch (e) {
             println('Error in the request')
         }
-        if (configs != null) {
+        if (configs != null && configs != 'null') {
```

Comment:
AFAIK in Groovy a string can be checked as a condition e.g. if (configs), and returns `false` if it's an empty string/null.